use-cases/vehicle-routing/src/main/resources/META-INF/resources/app.js

```diff
@@ -336,6 +337,116 @@ function renderTimelines(routePlan) {
     }
 }
 
+function analyze() {
+    new bootstrap.Modal("#scoreAnalysisModal").show()
+    const scoreAnalysisModalContent = $("#scoreAnalysisModalContent");
+    scoreAnalysisModalContent.children().remove();
+    if (loadedRoutePlan.score == null || loadedRoutePlan.score.indexOf('init') != -1) {
+        scoreAnalysisModalContent.text("Score not ready for analysis, try to run the solver first or wait until it advances.");
+    } else {
+        $.put("/route-plans/analyze", JSON.stringify(loadedRoutePlan), function (scoreAnalysis) {
```

Comment:
Consistency: we should picked either the `function (...) { }` or `(...) => { }` syntax, but not mix both of them.

Reply:
Agreed, will fix.

Reply:
Done.